tests/python/relax/test_frontend_nn_packing.py

```diff
@@ -25,7 +25,9 @@ def _iter_binding_names(mod):
     """Helper function to compare the names of relax variables"""
     for block in mod["forward"].body.blocks:
         for binding in block.bindings:
-            yield binding.var.name_hint
+            # Relax variable names may contain '.' even though it
+            # cannot be expressed in TVMScript.
```

Comment:
I wonder if this is something we should just check for and prohibit.

Reply:
I could go either way.  It's nice to have the 1:1 mapping between Relax and TVMScript, which would forbid the period within a relax variable name.  However, it's also nice to have a 1:1 mapping between a Relax function parameter and weight tensor's name in a pytorch or safetensor file, which are usually written with a period in the name.